portal/v2/src/index.tsx

```diff
@@ -14,4 +14,19 @@ mergeStyles({
   },
 })
 
-ReactDOM.render(<App />, document.getElementById("root"))
+const searchParams = window.location.search
+
+const getURLParams = () : any => {
+  let urlParams = {}
+  let paramStringFromURL = searchParams.split('?')[1];
+  let paramsArr = paramStringFromURL.split('&');
+
+  for (let i = 0; i < paramsArr.length; i++) {
+      let pair = paramsArr[i].split('=');
+      urlParams = {[pair[0]]: pair[1]}
+  }
+
+  return urlParams
+}
+
+ReactDOM.render(<App params={ searchParams ? getURLParams(): undefined} />, document.getElementById("root"))
```

Comment:
Please add a newline at the end of this file.

Reply:
Added a new line